tests/fixtures/MakeMessengerMiddleware/src/Middleware/CustomMiddleware.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Middleware;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Middleware\MiddlewareInterface;
+use Symfony\Component\Messenger\Middleware\StackInterface;
+
+final class CustomMiddleware implements MiddlewareInterface
+{
+    public function handle(Envelope $envelope, StackInterface $stack): Envelope
+    {
+        // ...
+        return $stack->next()->handle($envelope, $stack);
+    }
+}
```

Comment:
This file (and directory) aren't needed. fixtures directories are used when you need to *start* the test project with files *before* running the maker commands. It's not used as an "assert" that the created file matches this (or anything like that).